### PR TITLE
ICRC-95: Derivation origin standard

### PR DESCRIPTION
**Pull request by dostro**

### CODE REVIEW

File: topics/icrc_95_derivationorigin.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+### Definitions
+
+Signer: A user agent that manages keys and facilitates transactions with ICP.
+
+Relying party: A web page that relies upon one or many Web3 platform APIs which are exposed to the web page via the signer.
+
+### derivationOrigin Parameter
```
Comment (frederikrothenberger) [inline topics/icrc_95_derivationorigin.md:38]: Could you add a validation section / link to the II spec where the validation is explained?

Comment (sea-snake) [reply topics/icrc_95_derivationorigin.md:38]: @dostro That's the following link: https://internetcomputer.org/docs/current/references/ii-spec#alternative-frontend-origins

File: topics/icrc_49_call_canister.md [hunk 2]
```diff
@@ -106,9 +106,10 @@ An ICRC-25 compliant signer must implement the [icrc25_supported_standards](./ic
...
-        - The sender must make sure that the request complies with additional scope restrictions defined by the signer (if any), such as limitations on the target canister id or the sender principal, etc.
-3. The signer tries to retrieve and verify the consent message according to the [ICRC-21](icrc_21_consent_msg.md) specification. If the target canister does not support ICRC-21 consent messages for the given canister call, the signer may proceed in one of following ways:
+    - If the relying party has been denied the permission to invoke the method,
+      the signer sends a response with an error back to the relying party.
+      If the permission state is `ask_on_use`, the signer must prompt the user to either accept or deny this invocation. See [permission states](icrc_25_signer_interaction_standard.md#permissions-states) for more details.
+3. The signer tries to retrieve and verify the consent message according to the [ICRC-21](ICRC-21/icrc_21_consent_msg.md) specification. If the target canister does not support ICRC-21 consent messages for the given canister call, the signer may proceed in one of following ways:
```
Comment (sea-snake) [inline topics/icrc_49_call_canister.md:112]: Could you make sure other files are not changed by this MR? (besides readme overview list)

**Review by sea-snake — CHANGES_REQUESTED** [review]

**Review by sea-snake — APPROVED** [review]
Looks good to me!

**Pull request merged**